### DISCUSSION TITLE
Potentially illegal log file generation for bug reporting

### DISCUSSION THREAD

**Discussion by StefanKarlsson987** [mpv-player/mpv#14574] [General]
great player in so many ways!

I just submitted a first bug report, and posted the output.txt file to the forum.
It occured to me that I should have a look in the file. It turns out that it contains personal information of the user. This includes lists of recent files that the user have viewed with the player. 

I found no information that I did not want to share in the file so I went ahead and uploaded it.
I dont know the legal implications, but I believe many users would feel this is not information they want to share without giving consent to it. To avoid annoying users, avoid possible legal difficulties, I would suggest that the output.txt information be scrubbed. Perhaps this can be done by adding a command-line option

**Comment by llyyr** [comment]
That information is printed by scripts you're using, we have no control over what scripts you decide to use with mpv. Moreover, the template asks you to provide a log with --no-config which would've prevented this useless information being added to the log file.

**Comment by kasper93** [comment]
> I dont know the legal implications

So, why are mentioning it, if you don't know what you are talking about?

> I believe many users would feel this is not information they want to share without giving consent to it. 

You created a file, text file, uploaded it to github, yourself, without any external force. Who do you want to give consent for your own action?

We can add declaimer that log file may contain certain "personal" information like GPU model, but this is kinda implied the log file contains a lot information and you should always think what you send to the "internet".
